src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -43,24 +50,32 @@ public partial class CryptoStream : System.IO.Stream, System.IDisposable
         public override long Position { get { return default(long); } set { } }
         protected override void Dispose(bool disposing) { }
         public override void Flush() { }
-        public override System.Threading.Tasks.Task FlushAsync(System.Threading.CancellationToken cancellationToken) { return default(System.Threading.Tasks.Task); }
```

Comment:
In general we don't enforce that a member is overridden but we do try to keep the overridden member in the ref if we know of any implementations that have the override.